go.mod

```diff
@@ -36,6 +36,8 @@ replace github.com/99designs/keyring => github.com/crypto-org-chain/keyring v1.1
 replace github.com/cosmos/ibc-go => github.com/crypto-org-chain/ibc-go v1.0.1-hooks
 
 // FIXME: update after PR merged: https://github.com/PeggyJV/gravity-bridge/pull/182
-replace github.com/peggyjv/gravity-bridge/module => github.com/crypto-org-chain/gravity-bridge/module v0.1.22-0.20210923075944-630e7235f4b2
+replace github.com/peggyjv/gravity-bridge/module => github.com/yihuang/gravity-bridge/module v0.1.22-0.20211003003616-466845e76fcf
 
 replace github.com/cosmos/iavl => github.com/cosmos/iavl v0.17.1
+
+replace github.com/tharsis/ethermint => github.com/yihuang/ethermint v0.4.2-0.20211003002300-c53ebc8bbdaf
```

Comment:
which version of ethermint does it uses?

Reply:
the exact commit in this PR: https://github.com/tharsis/ethermint/pull/615

Reply:
can you replace it by `crypto.org/ethermint` ? It would be nice also to tag the version we wants to patch